server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultClusterContext.java

```diff
@@ -125,6 +125,12 @@ public DefaultClusterContext(String name,
         this.clusterMembers.addAll(members);
         this.schedulerProvider = schedulerProvider;
         this.connectOptions = connectOptions;
+
+        if (!GL.isTraceEnabled(GL.CLUSTER_LOGGER_NAME)) {
+            // Note: must use Logger.getLogger, not LogManager.getLogger
```

Comment:
we were always disabling this in the past with no way of enabling it without making a code change, I wanted to make it so it is enabled as part of HA trace.